Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+
+--
+-- TOC entry 227 (class 1259 OID 17859)
+-- Name: fittedFluxStatistics; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedFluxStatistics" (
+    id integer NOT NULL,
+    simulation_id text,
+    "simulation_dateAndTime" timestamp(6) without time zone,
+    n_fluxes integer,
+    "n_observableFluxes" integer,
+    total_precision double precision,
+    "total_observablePrecision" double precision,
+    "relative_nObservableFluxes" double precision,
+    "average_observableFluxPrecision" double precision,
+    "average_fluxPrecision" double precision,
+    "flux_units " text,
+    used boolean,
+    comment text
+);
+
+
+ALTER TABLE public."fittedFluxStatistics" OWNER TO postgres;
+
+--
+-- TOC entry 247 (class 1259 OID 17974)
+-- Name: fittedFluxStatistics_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."fittedFluxStatistics_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."fittedFluxStatistics_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3145 (class 0 OID 0)
+-- Dependencies: 247
+-- Name: fittedFluxStatistics_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."fittedFluxStatistics_id_seq" OWNED BY public."fittedFluxStatistics".id;
+
+
+--
+-- TOC entry 228 (class 1259 OID 17862)
+-- Name: fittedFluxes; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedFluxes" (
+    id integer NOT NULL,
+    simulation_id text,
+    "simulation_dateAndTime" timestamp(6) without time zone,
+    " rxn_id" text,
+    flux double precision,
+    "flux_stdev " double precision,
+    flux_lb double precision,
+    flux_ub double precision,
+    flux_units text,
+    fit_alf double precision,
+    fit_chi2s double precision[],
+    fit_cor double precision[],
+    fit_cov double precision[],
+    free boolean,
+    used boolean,
+    comment text
+);
+
+
+ALTER TABLE public."fittedFluxes" OWNER TO postgres;
+
+--
+-- TOC entry 248 (class 1259 OID 17985)
+-- Name: fittedFluxes_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."fittedFluxes_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."fittedFluxes_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3146 (class 0 OID 0)
+-- Dependencies: 248
+-- Name: fittedFluxes_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."fittedFluxes_id_seq" OWNED BY public."fittedFluxes".id;
+
+
+--
+-- TOC entry 229 (class 1259 OID 17865)
+-- Name: fittedFragments; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedFragments" (
+    id integer NOT NULL,
+    simulation_id text,
+    "simulation_dateAndTime" timestamp(6) without time zone,
+    experiment_id text,
+    " sample_name_abbreviation" text,
```

Comment:
Please remove this extra whitespace and those below.

Reply:
ddone